src/components/Form.js

```diff
@@ -66,24 +79,38 @@ export default function Form({ handleActivity, preloadedValues, title }) {
         notes: data.notes,
         date: data.date,
         location: data.location,
+        photo: photo,
       });
       navigate(`/`);
+      setPhoto('');
     }
   };
 
   useEffect(() => {
     setFocus('activity');
   }, [setFocus]);
 
+  function onDeletePicture(event) {
+    event.preventDefault();
+    setPhoto('');
+  }
+  const [preloadedPicture, setPreloadedPicture] = useState(
```

Comment:
states should always be at the beginning of the function declared ;)

Reply:
haha you are right, this was actually the plan, just had it here for overview reasons shortly - or in this case longer😅